tests/Database/DatabaseQueryBuilderTest.php

```diff
@@ -873,7 +901,7 @@ public function testUnionAggregate()
         $builder->getProcessor()->shouldReceive('processSelect')->once();
         $builder->from('posts')->select('id')->union($this->getMySqlBuilder()->from('videos')->select('id'))->count();
 
-        $expected = 'select count(*) as aggregate from (select * from "posts" union select * from "videos") as "temp_table"';
+        $expected = 'select count(*) as aggregate from ((select * from "posts") union (select * from "videos")) as "temp_table"';
```

Comment:
This PR now adds them always, even if not necessary. But I guess this is fine? 🤷‍♀️

Reply:
Yes, just only a select clause is not necessary, but if a `LIMIT` or `GROUP BY` clause is added, parentheses are necessary in Postgres. I guess that this behavior is not a problem. 😄